code/iaas/allocator/src/main/java/io/cattle/platform/allocator/dao/impl/AllocatorDaoImpl.java

```diff
@@ -1,28 +1,33 @@
 package io.cattle.platform.allocator.dao.impl;
 
-import static io.cattle.platform.core.model.tables.AgentTable.*;
```

Comment:
I dont want to merge with this formatting change. Can you change it back? There is a setting in eclipse that controls how many static imports will cause a .* can you change that setting to 1?

Preferences > Java > Organize Imports > Number of imports needed for .*